UI/Panels/LogPanel.cs

```diff
@@ -1,24 +1,31 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Drawing;
-using System.Linq;
 using System.Reflection;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
 
 namespace MobiFlight.UI.Panels
 {
     public partial class LogPanel : UserControl, ILogAppender
     {
+        private bool IsDrawingSuspended = false;
+        private System.Windows.Forms.Timer CheckDrawingSuspendedTimer = new Timer();
         public LogPanel()
         {
             InitializeComponent();
             listView1.Items.Clear();
             listView1.Resize += ListView1_Resize;
             listView1.DoubleBuffered(true);
+            CheckDrawingSuspendedTimer.Interval = 50;
```

Comment:
Any reason you picked 50ms? That's still way faster than any human could process a log event, why not go with 250ms (to pick another random number :)

Reply:
Yes. Any slower with a delay like 100ms or 200ms did not improve cpu or lag any further. But user experience is a bit less nice. And faster makes no sense, because the maximum accuracy of windows forms timer is around 55ms.